code/src/main/kotlin/com/expediagroup/sdk/core/client/AbstractRequestExecutor.kt

```diff
@@ -52,7 +55,7 @@ import com.expediagroup.sdk.core.model.exception.service.ExpediaGroupNetworkExce
  *
  * @param transport The transport implementation to use for executing requests
  */
-abstract class RequestExecutor(protected val transport: Transport) {
+abstract class AbstractRequestExecutor(protected val transport: Transport) : Disposable {
```

Comment:
Out of curiosity, why did we rename `RequestExecutor` to `AbstractRequestExecutor`?

Reply:
Discussed offline ✅